.golangci.yml

```diff
@@ -0,0 +1,10 @@
+linters:
+  enable:
+    - errcheck
+    - gosimple
+    - govet
+    - ineffassign
+    - staticcheck
+    - unused
```

Comment:
A few things that might be nice in this file:
* indicating which linters are default (probably unobjectionable/ordinary) and which ones we're specifically enabling
* a link to the ref docs.